.travis.yml

```diff
@@ -1,48 +1,60 @@
+sudo: false
+git:
+  depth: 3
 matrix:
+  fast_finish: true
   include:
-  - name: "DApp unit tests"
```

Comment:
I don't see those being run in the new version of this file. But we probably want to keep those DApp unit tests since they run the linter which is useful to enforce consistent coding style for the DApp ?